electron/main.js

```diff
@@ -340,11 +340,23 @@ app.on('ready', function() {
 ///////////////////////////////////////////////////////////////////////////////
 // Delete the console.log
 ///////////////////////////////////////////////////////////////////////////////
-let consoleLog = path.join(app.getPath('userData'), config.CONSOLE_LOG);
-fs.stat(consoleLog, function(err, stats) {
-  if (!err) {
-    fs.rename(consoleLog, consoleLog.replace('.log', '.old'));
+const logDir = path.join(app.getPath('userData'), 'logs');
+fs.readdir(logDir, (error, files) => {
+  if (error) {
+    console.log(`Failed to read log directory ${error.message}`)
```

Comment:
"Failed to read log directory**:**" to put a seperation between our log message and the log message of others.